glidertest/tools.py

```diff
@@ -194,51 +194,55 @@ def plot_basic_vars(ds, v_res=1, start_prof=0, end_prof=-1):
     [a.set_ylim(depthG.max() + 10, -5) for a in ax]
     [a.grid() for a in ax]
 
-
-def chl_first_check(ds):
+def optics_first_check(ds, var='CHLA'):
     """
-    Function to asses any drift in deep chlorophyll data and the presence of any possible negative data
+    Function to asses any drift in deep optics data and the presence of any possible negative data
     This function returns plots and text
     """
     
     # Check how much negative data there is
-    neg_chl = np.round((len(np.where(ds.CHLA < 0)[0]) * 100) / len(ds.CHLA), 1)
+    neg_chl = np.round((len(np.where(ds[var] < 0)[0]) * 100) / len(ds[var]), 1)
     if neg_chl > 0:
-        print(f'{neg_chl}% of scaled chlorophyll data is negative, consider recalibrating data')
+        print(f'{neg_chl}% of scaled {var} data is negative, consider recalibrating data')
         # Check where the negative values occur and if we just see them at specific time of the mission or not
-        start = ds.TIME[np.where(ds.CHLA < 0)][0]
-        end = ds.TIME[np.where(ds.CHLA < 0)][-1]
-        min_z = (np.round(ds.DEPTH[np.where(ds.CHLA < 0)].min().values, 1))
-        max_z = (np.round(ds.DEPTH[np.where(ds.CHLA < 0)].max().values, 1))
+        start = ds.TIME[np.where(ds[var] < 0)][0]
+        end = ds.TIME[np.where(ds[var] < 0)][-1]
+        min_z = ds.DEPTH[np.where(ds[var] < 0)].min().values
+        max_z = ds.DEPTH[np.where(ds[var] < 0)].max().values
         print(f'Negative data in present from {str(start.values)[:16]} to {str(end.values)[:16]}')
-        print(f'Negative data is present between {min_z} and {max_z} ')
+        print(f'Negative data is present between {"%.1f" % np.round(min_z, 1)} and {"%.1f" % np.round(max_z, 1)} ')
     else:
         print('There is no negative scaled chlorophyll data, recalibration and further checks are still recommended ')

```

Comment:
scaled chlorophyll >> var

good to make the function more generic! Make sure to update all the prints like this one to reflect it